x11rb/examples/simple_window.rs

```diff
@@ -113,7 +124,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     loop {
         let event = conn.wait_for_event()?;
-        println!("{:?})", event);
+        tracing::event!(tracing::Level::DEBUG, "Got event {event:?}");
```

Comment:
Does the library have shortcut macros (e.g. 'tracing::debug!')?

Reply:
Yup. Changed this code to those macros.

I started with `simple_window.rs` to learn how to use `tracing` and "did not look back" at the end. Thanks for catching that!